roles/kubernetes/cri/containerd/defaults/main.yaml

```diff
@@ -0,0 +1,6 @@
+# Copyright © 2022-2023, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Allows users to overwrite containerd version
+# Set as an empty string to use the latest upstream version
+containerd_version: "1.6.20-1"
```

Comment:
Link or reference here to version information and releases.

Reply:
I put the detailed steps on how to find packages here as well